### PR TITLE
#8764: Various docs fixes for WH release

### PR DESCRIPTION
**Pull request by tt-rkim**
Add disclaimer that current WH models are only on N300 and fix… installing link in docs

### CODE REVIEW

File: docs/source/ttnn/tt_metal_models/get_started.rst [hunk 1]
```diff
@@ -10,12 +10,12 @@ Ensure that you have the base TT-Metalium source and environment configuration
...
 
-Now, from the project root, get the Python virtual environment in which you'll
-be working in ready.
+Now, from the project root, activate the provided Python virtual environment in
+which you'll be working.
 
 ::
 
-    source build/python_env/bin/activate
+    source python_env/bin/activate
```
Comment (TT-BrianLiu) [inline docs/source/ttnn/tt_metal_models/get_started.rst:18]: There's an env variable (`PYTHON_ENV_DIR`) that can change where this is. Maybe make a note of that here?

File: docs/source/tt-metalium/get_started/get_started.rst [hunk 2]
```diff
@@ -17,59 +17,36 @@ Quick Start Guide
 Metalium provides developers to do more than running models, facilitating a
 transition from running models effortlessly out of the box, engaging in
 lightweight optimizations, and progressing into more sophisticated, heavyweight
-optimizations. This series of five steps serves as an illustrative example,
+optimizations. This series of steps serves as an illustrative example,
```
Comment (TT-BrianLiu) [inline docs/source/tt-metalium/get_started/get_started.rst:20]: Grammar: `serve`

**Review by TT-BrianLiu — APPROVED** [review]

File: docs/source/tt-metalium/get_started/get_started.rst (hunk 2 above)
Comment (TT-BrianLiu) [reply docs/source/tt-metalium/get_started/get_started.rst:20]: Nvm im clowning

File: docs/source/ttnn/tt_metal_models/get_started.rst (hunk 1 above)
Comment (tt-rkim) [reply docs/source/ttnn/tt_metal_models/get_started.rst:18]: Done, pls take a look

**Pull request merged**